test/powershell/engine/Remoting/RemoteSession.Basic.Tests.ps1

```diff
@@ -1,11 +1,14 @@
-
 Describe "New-PSSession basic test" -Tag @("CI") {
+    if ( ! $IsWindows ) {
+        $PSDefaultParameterValues['it:skip'] = $true
+    }
     It "New-PSSession should not crash powershell" {
         try {
-            New-PSSession -ComputerName nonexistcomputer -Authentication Basic
```

Comment:
Please don't change formatting. This complicates the code review. A formatting must be in special PRs.